src/App.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+    <Route path="/" exact component={Home} />
+    <Route path="/about" component={About}/>
+    <Route path="/contact" component={Contact}/>
```

Comment:
excellent, good catching that you don't need to specify About and Contact with the `exact` path here, only Home